frontends/election-manager/src/app.test.tsx

```diff
@@ -602,17 +636,9 @@ test('tabulating CVRs', async () => {
 
   fireEvent.click(getByText('Close'));
 
-  // Check that a zero report can be generated on the L&A tab even after CVRs have been tabulated
+  // Check L&A Materials are unavailable after live CVRs have been loaded
```

Comment:
Again, great catch that the old behavior wasn't desirable from a product perspective!